acceptance-tests/xfcc_test.go

```diff
@@ -293,3 +351,32 @@ func parseXFCCHeader(header string) string {
 
 	return certPEM.String()
 }
+
+func parseXFCCHeaderToCert(header string) *x509.Certificate {
+	derEncodedCert, err := base64.StdEncoding.DecodeString(header)
+	Expect(err).NotTo(HaveOccurred())
+
+	cert, err := x509.ParseCertificate(derEncodedCert)
+	Expect(err).NotTo(HaveOccurred())
+
+	return cert
+}
+
+func checkXFCCHeaders(recordedXFCCHeader string, recordedHeaders http.Header) {
+	cert := parseXFCCHeaderToCert(recordedXFCCHeader)
+	Expect(recordedHeaders.Get("X-Ssl-Client")).To(Equal("1"))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Verify")).To(Equal("0"))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.Country[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.Organization[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Dn")).To(ContainSubstring(cert.Subject.CommonName))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Subject-Cn")).To(Equal(cert.Subject.CommonName))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.Country[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.Organization[0]))
+	Expect(recordedHeaders.Get("X-Ssl-Client-Issuer-Dn")).To(ContainSubstring(cert.Issuer.CommonName))
+	notBeforeString := fmt.Sprintf("%02d%02d%02d%02d%02d%02dZ", cert.NotBefore.Year()-2000,
+		cert.NotBefore.Month(), cert.NotBefore.Day(), cert.NotBefore.Hour(), cert.NotBefore.Minute(), cert.NotBefore.Second())
```

Comment:
The not-before and not-after time comparisons could be simplified using the "magic" [Go Reference Time](https://golang.org/pkg/time/#Time.Format) on January 2nd 2006, as:
```
Expect(recordedHeaders.Get("X-Ssl-Client-Notbefore")).To(Equal(cert.NotBefore.Format("060102150405Z")))
```

(you might need `cert.NotBefore.UTC()` instead)

Reply:
done